content/tutorials/presence.textile

```diff
@@ -137,110 +206,145 @@ blang[ruby].
+You are now in the presence set. The name you will appear as is the @client ID@ that you used to instantiate the Ably client library in step 2.
 
-We're now in the presence set. The name we'll appear as is the @client ID@ that we included in the Realtime constructor in step 2.
+Similarly, you can call @channel.presence.leave@ to leave the presence set.
 
-Similarly, we can call @channel.presence.leave@ to leave the presence set.
-
-(Side note: while in this tutorial we had free choice of @client ID@, in most situations you don't want that. Instead you want your server to constrain what @client ID@ people can use -- for example, to their logged-in username -- so that no-one can pretend to be someone else. When using "Token authentication":/core-features/authentication#token-authentication, your server can issue tokens with a specific @client ID@. When such a token is used by someone, their connection is bound to that @client ID@.)
+(Side note: while in this tutorial you can choose any @client ID@, in most situations you don't want that. Instead you want your server to constrain what @client ID@ people can use -- for example, to their logged-in username -- so that no-one can pretend to be someone else. When using "Token authentication":/core-features/authentication#token-authentication, your server can issue tokens with a specific @client ID@. When such a token is used by someone, their connection is bound to that @client ID@.)
 
 A single @client ID@ may be present multiple times on the same channel, via different client connections; but each will be seen as a different member of the presence set, as they are differentiated by their unique @connection ID@. For example, if a client with @client ID@ “Sarah” is connected to a chat channel on both a desktop and a mobile device simultaneously, “Sarah” will be present twice in the presence member set with the same client ID, each with a different @connection ID@. A member of the presence set is therefore unique by the combination of the @client ID@ and @connection ID@ strings.
 
-h3. Step 3a - Updating your presence data
+h3. Updating your presence data (optional)
 
-In the previous step, we set a payload for our entry in the presence set: the string 'hello', which other clients will receive with the notification that we've entered. We can update that whenever we want, and other clients will receive the update:
+In the previous step, you set a payload for your entry in the presence set: the string "hello", which other clients will receive with the notification that this particular client has entered the channel. You can update that whenever you want, and other clients will receive the update. This is not required for the tutorial, but the following code shows how to achieve it:
 
 blang[javascript,nodejs].
   ```[javascript]
-    channel.presence.update('howdy!', function(err) {
+    channel.presence.update("howdy!", (err) => {
       if(err) { return console.error("Error updating presence data"); }
```

Comment:
as a single line the `if` does not requie `{...}` scope.

Reply:
Done